### PR TITLE
Remove default font color from global CSS to enable overwrites

### PR DESCRIPTION
**Pull request by huong-li-nguyen**
## Description

Bug: Font color in data table could not be updated via `style_data_conditional`

- Removes default font color from global CSS to enable overwrites
- Assigns local font color in relevant CSS classes

## Screenshot

Screenshot 2023-12-13 at 18 50 57

## Checklist

- [x] I have not referenced individuals, products or companies in any commits, directly or indirectly
- [x] I have not added data or restricted code in any commits, directly or indirectly
- [x] I have updated the docstring of any public function/class/model changed
- [ ] I have added tests to cover my changes (if applicable)

## Types of changes

- [ ] Docs/refactoring (non-breaking change which improves codebase)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

  - I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
  - I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorized to submit this contribution on behalf of the original creator(s) or their licensees.
  - I certify that the use of this contribution as authorized by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

### CODE REVIEW

File: vizro-core/examples/default/app.py [hunk 1]
```diff
@@ -427,6 +427,7 @@ def create_country_analysis():
                 figure=dash_data_table(
                     id="dash_data_table_country",
                     data_frame=px.data.gapminder(),
+                    style_cell_conditional=[{"if": {"column_id": "pop"}, "color": "orange"}],
```
Comment (huong-li-nguyen) [inline vizro-core/examples/default/app.py:430]: Just for demo purposes - I'll remove it afterwards

File: vizro-core/src/vizro/static/css/accordion.css [hunk 2]
```diff
@@ -26,24 +26,25 @@
...
-  font-size: var(--text-size-02);
-  line-height: 16px;
-  position: relative;
-  display: flex;
   align-items: center;
-  width: 100%;
-  text-align: left;
-  overflow-anchor: none;
   background-color: inherit;
+  color: var(--text-secondary);
```
Comment (huong-li-nguyen) [inline vizro-core/src/vizro/static/css/accordion.css:37]: Only the color attribute was added - the rest is just linting

File: vizro-core/src/vizro/static/css/index.css [hunk 3]
```diff
@@ -11,7 +11,6 @@
 * {
   border: 0;
   box-sizing: border-box;
-  color: var(--text-secondary);
```
Comment (huong-li-nguyen) [inline vizro-core/src/vizro/static/css/index.css:14]: This is the important change

File: vizro-core/src/vizro/static/css/token_names.css [hunk 4]
```diff
@@ -1,85 +1,85 @@
 :root {
-  --state-overlays-dark-mode-enable: rgba(255, 255, 255, 0);
```
Comment (huong-li-nguyen) [inline vizro-core/src/vizro/static/css/token_names.css:2]: No changes, just linting

**Review by maxschulz-COL — APPROVED** [review]
Lgtm! Hope it wasn't too bad for finding all these cases...

**Review by nadijagraca — APPROVED** [review]
Looks good. Thanks for fixing it. 🚀

**Pull request merged**